docs/docs/guides/migrate/users.md

```diff
@@ -0,0 +1,231 @@
+Please also consult our [guide](/docs/guides/manage/user/reg-create-user) on how to create users.
+
+```json
+{
+  "userName": "test9@test9",
+  "profile": {
+    "firstName": "Road",
+    "lastName": "Runner",
+    "displayName": "Road Runner",
+    "preferredLanguage": "en"
+  },
+  "email": {
+    "email": "test@test.com",
+    "isEmailVerified": false
+  },
+  "hashedPassword": {
+    "value": "$2a$14$aPbwhMVJSVrRRW2NoM/5.esSJO6o/EIGzGxWiM5SAEZlGqCsr9DAK",
+    "algorithm": "bcrypt"
+  },
+  "passwordChangeRequired": false,
+  "otpCode": "testotp",
+  "requestPasswordlessRegistration": false,
+  "idps": [
+    {
+      "configId": "124425861423228496",
+      "externalUserId": "roadrunner@mailonline.com",
+      "displayName": "name"
+    }
+  ]
+}
+```
+
+## Bulk import
+
+For bulk import use the [import endpoint](https://zitadel.com/docs/apis/admin/admin-service-import-data) on the admin API:
+
+```json
+{
+  "timeout": "10m",
+  "data_orgs": {
+    "orgs": [
+      {
+        "orgId": "104133391254874632",
+        "org": {
+          "name": "TestFabi"
+        },
+        "humanUsers": [
+          {
+            "userId": "104133391271651848",
+            "user": {
+              "userName": "fabienne",
+              "profile": {
+                "firstName": "Fabienne",
+                "lastName": "Gerschwiler",
+                "displayName": "Fabienne Gerschwiler",
+                "preferredLanguage": "und"
+              },
+              "email": {
+                "email": "fabienne@caos.ch",
+                "isEmailVerified": true
+              },
+              "hashedPassword": {
+                "value": "$2a$14$aPbwhMVJSVrRRW2NoM/5.esSJO6o/EIGzGxWiM5SAEZlGqCsr9DAK",
+                "algorithm": "bcrypt"
+              }
+            }
+          },
+          {
+            "userId": "120080115081209416",
+            "user": {
+              "userName": "testuser",
+              "profile": {
+                "firstName": "Test",
+                "lastName": "User",
+                "displayName": "Test User",
+                "preferredLanguage": "und"
+              },
+              "email": {
+                "email": "fabienne@caos.ch",
+                "isEmailVerified": true
+              },
+              "hashedPassword": {
+                "value": "$2a$14$785Fcdbpo9rn5L7E21nIAOJvGCPgWFrZhIAIfDonYXzWuZIKRAQkO",
+                "algorithm": "bcrypt"
+              }
+            }
+          },
+          {
+            "userId": "145195347319252359",
+            "user": {
+              "userName": "test1",
+              "profile": {
+                "firstName": "Fabienne",
+                "lastName": "Gerschwiler",
+                "displayName": "Fabienne Gerschwiler",
+                "preferredLanguage": "und"
+              },
+              "email": {
+                "email": "fabienne@gmail.com"
```

Comment:
We should at least change that because its not my email and we don't want to spam a random person in case someone uses this data. 

```suggestion
                "email": "testuser@gmail.com"
```

Reply:
Makes sense. Changed it do dummies.